cli/src/main.rs

```diff
@@ -60,7 +62,11 @@ async fn main() -> color_eyre::Result<()> {
             allow_existing,
         } => {
             let user_id = get_user_id_from_session()?;
-            append_url_to_db(&db_pool, &url, user_id, allow_existing).await?;
+            append_url(&db_pool, &url, user_id, allow_existing).await?;
+            // let outcome = append_url(&db_pool, &url, user_id, allow_existing).await?;
+            // if let AddUrlOutcome::Created(page) | AddUrlOutcome::Existing(page) = outcome {
+            //     process_page_snapshot(&db_pool, page.page_id, user_id, &url).await?;
```

Comment:
The `page` within `outcome` is the 'full' Page row right?

Like it has a `page.url` that we could use right? It might be nicer to pass the full `page` down to `process_page_snapshot` so it doesn't need to fetch the URL by id right away!